community-roles.md

```diff
@@ -113,6 +114,23 @@ The following apply to all assets across the Flux org:
 - Defining the scope of each Git repository
 - Resolving escalated decisions when Maintainers responsible are blocked
 
+### Security Team member
+
+Security Team members are listed in the [SECURITY.md file](https://github.com/fluxcd/.github/blob/main/SECURITY.md#security-team).
+
+Members of this team handle security issues for the Flux projects. It is essential for us to deal with security-related concerns responsibly and follow the high standards of the security community as a whole.
+
```

Comment:
Although https://github.com/fluxcd/.github/pull/19 hasn't been merged yet, I think we should highlight that Security Team members have a duty of care to the uphold of the Security embargo policy.